From d67184761996908a5d8d7c7c773017257cdfced2 Mon Sep 17 00:00:00 2001
Message-ID: <d67184761996908a5d8d7c7c773017257cdfced2.1694125610.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1694125609.git.sharlatanus@gmail.com>
References: <cover.1694125609.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 7 Sep 2023 21:32:03 +0100
Subject: [PATCH 14/19] gnu: python-rad: Update to 0.17.1.

* gnu/packages/astronomy.scm (python-rad): Update to 0.17.1.
[arguments]{tests?}: Disable them to break dependencies cycle.
[native-inputs]: Remove python-astropy, python-pytest,
python-pytest-doctestplus, python-pytest-openfiles.
---
 gnu/packages/astronomy.scm | 16 +++++++---------
 1 file changed, 7 insertions(+), 9 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 95c585079c..a402c63e9f 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3807,22 +3807,20 @@ (define-public python-gwcs
 (define-public python-rad
   (package
     (name "python-rad")
-    (version "0.15.0")
+    (version "0.17.1")
     (source (origin
               (method url-fetch)
               (uri (pypi-uri "rad" version))
               (sha256
                (base32
-                "0j51pkywxdaqrfz162rdsywlvx1mbb2h0gi5framvhf25i1im7mb"))))
+                "11jhh3qmd00g8sn6hcfshbpb2qy1rfj0xkxn30pd63lqazg4ra3p"))))
     (build-system pyproject-build-system)
     (arguments
-     (list #:test-flags #~(list "-k" "not remote_data")))
-    (native-inputs (list python-astropy
-                         python-pytest
-                         python-pytest-doctestplus
-                         python-pytest-openfiles
-                         python-semantic-version
-                         python-setuptools-scm))
+     (list
+      ;; XXX: Tests introduce dependencies cycle, breaking it by disable them.
+      ;; python-rad -> python-roman-datamodels -> python-crds -> python-rad
+      #:tests? #f))
+    (native-inputs (list python-semantic-version python-setuptools-scm))
     (propagated-inputs (list python-asdf python-asdf-astropy))
     (home-page "https://github.com/spacetelescope/rad")
     (synopsis "Roman Attribute Dictionary")
-- 
2.41.0

